.circleci/config.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
```

Comment:
I setup the circleci build to run the tests, but it's missing the deployment key to be able to checkout the project, can you help me with this @raivil or only Daniel can do it?

Reply:
@marcocarvalho I started "following" the repository on CircleCI and it looks like it added a deploy key automatically on GitHub.
```
Added on Jan 18, 2022 by CircleCI with authorization from @raivil
Never used — Read-only
```

Re-running the project works. ✨